numpy/core/src/multiarray/calculation.c

```diff
@@ -1186,6 +1187,13 @@ PyArray_Conjugate(PyArrayObject *self, PyArrayObject *out)
     }
     else {
         PyArrayObject *ret;
+        if (!PyArray_ISNUMBER(self)) {
+            if (DEPRECATE("attempting to conjugate non-numeric dtype; "
+                  "this will error in the future to match the behavior "
+                  "of np.conjugate") < 0) {
```

Comment:
Indentation is weird here - should probably either be column aligned, or 8 spaces - currently, it's neither